### ISSUE TITLE
How can I use $activateValidator()  twice with lazy initialization?

### ISSUE THREAD

**Issue by jxiaox** [kazupon/vue-validator#122]
In my page ,I used one component to load create/update page.
so i need to activate validator again in this component,and i notice that $activateValidator() destroyed when calling this method, am I right? 
Or is there another way to do this? Thanks.

**Comment by kazupon** [comment]
Thank you for your feedback. :smile_cat: 

The `$activateValidator()` meta method is destroyed when once called.

The Lazy initialization of validator  with `$activateValidator()` is expected in once only initialization usecase (ex: ajax data fetching, after that these data setting vue instance).

> In my page ,I used one component to load create/update page.

However, there is something we a little curious about your usecase.
Would you tell us about it ?

**Comment by jxiaox** [comment]
Think's for your answer.
In my case,I use one page to show create and update.First,I show create page to create new page;And then I have a list to enter update page,and in update page user can modify it.Because those tow page has same form input,so I think I can use one page to show it.

**Comment by kazupon** [comment]
Thank you for your feedback.
I want to understand about your use-case more.
Could you provide the minimum example codes ?

**Comment by lazabogdan** [comment]
@kazupon I've just hit a wall on this as well. 

For a simple use case example, just imagine two validators attached to the same vm, one lazy validator and one **not lazy** and the 2nd validator is inside a v-if directive which only activates after you've loaded the data for the lazy validator and called $activateValidator. 

Also there are more possible scenarios for which you might need to activate a validator more than once in the same vm.

So basically it's an issue when you need more than one validator on the same vm.

Another example is having multiple modals on the same vm and each having it's own validator.

Let me know if you need more details.

**Comment by ikristher** [comment]
I also have a similar use case where in I need this feature. Hopefully this feature will be added soon

**Comment by kazupon** [comment]
I'll try to implement at v3.0.
